src/routes/projects/[id([0-9]+)]/index.svelte

```diff
@@ -355,6 +356,12 @@
 
 <svelte:head>
   <title>{project.name} – Innopoints</title>
+  <meta name="og:title" content={project.name} />
+  <meta name="og:url" content="https://ipts.innopolis.university/projects/{project.id}" />
+  <meta name="og:description" content="Organized by {project.organizer}" />
```

Comment:
Pretty sure we can fit in another line saying: _Available activities: X, Y, Z..._, at most three, less if we run out of space